frontend/src/Initialize.tsx

```diff
@@ -219,22 +219,25 @@ const Initialize = (props: Props) => {
 
   const saveGameSettings = (evt: any) => {
     evt.preventDefault();
-    localStorage.setItem('gameSettingsInLocalStorage', JSON.stringify(props.state.propagated));
+    localStorage.setItem(
+      "gameSettingsInLocalStorage",
+      JSON.stringify(props.state.propagated)
+    );
   };
 
   const loadGameSettings = (evt: any) => {
     evt.preventDefault();
-    const settings = localStorage.getItem('gameSettingsInLocalStorage');
+    const settings = localStorage.getItem("gameSettingsInLocalStorage");
     if (settings !== null) {
       const gameSettings: IPropagatedState = JSON.parse(settings);
       for (const [key, value] of Object.entries(gameSettings)) {
         switch (key) {
-          case 'game_mode':
-            if (value === 'Tractor') {
+          case "game_mode":
```

Comment:
I think you can actually just do

```
send({Action:{SetGameMode: value}})
```
similar to the other cases? the format should be compatible on the wire.
```